File: src/main/java/com/ksptool/ql/biz/user/model/vo/GetGroupListVo.java
package com.ksptool.ql.biz.user.model.vo;

import java.util.Date;
import lombok.Data;

@Data
public class GetGroupListVo {

    // 组ID
    private Long id;

    // 组标识
    private String code;

    // 组名称
    private String name;

    // 成员数量
    private Integer memberCount;

    // 权限节点数量
    private Integer permissionCount;

    // 系统内置组
    private Boolean isSystem;

    // 组状态：0-禁用，1-启用
    private Integer status;

    // 创建时间
    private Date createTime;
}

File: src/main/resources/views/aether/src/entity/vo/GetGroupListVo.ts
// 组列表视图对象
export default interface GetGroupListVo {
    // 组ID
    id: string;

    // 组标识
    code: string;

    // 组名称
    name: string;

    // 成员数量
    memberCount: number;

    // 权限节点数量
    permissionCount: number;

    // 系统内置组
    isSystem: boolean;

    // 组状态：0-禁用，1-启用
    status: number;

    // 创建时间
    createTime: string;
} 